src/data/factions.ts

```diff
@@ -93,7 +93,11 @@ export function factionDesc(faction: Faction) {
     <b>Round Income:</b> ${roundIncome} </br>
     <span style="white-space: nowrap; line-height: 1em">
       <b>Steps:</b> 
-      ${[0,1,2,3].map(i => `<span class="ml-2">${planetsWithSteps(p, i).map(p => `<i class="planet ${p}"></i>`).join(" ")} ${i}</span>`).join(" ")}
+      ${[0,1,2,3].map(i => `<span class="ml-2">${planetsWithSteps(p, i).map(p => 
+        `<svg width="15" height="20" viewbox="0 0 15 15" >
+          <circle cx="7" cy="6" r="6"  class="player-token planet-fill ${p}" />
```

Comment:
Is it normal that cx/cy are different than in PlayerInfo (7,6 instead of 8,8)?

Reply:
Definitely css is not my best skill ;)
I had to do this because if not I was having 
<img width="338" alt="screen shot 2018-09-28 at 13 57 08" src="https://user-images.githubusercontent.com/29941924/46207149-9faeb280-c326-11e8-807d-263004e8bc1d.png">